modules/serverless-config-log-settings-serving-eventing.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * serverless/observability/serverless-config-log-settings.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-config-log-settings-serving-eventing_{context}"]
+= Configuring log settings
+
+.Procedure
+
+* Configure the log settings for Serving and Eventing by setting or modifying the `loglevel` value in the `KnativeServing` and `KnativeEventing` CR respectively. Here are two example configurations with all possible logging options set to level `info`:
++
+.KnativeServing CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  config:
+    logging:
+      loglevel.controller: "info"
+      loglevel.autoscaler: "info"
+      loglevel.queueproxy: "info"
+      loglevel.webhook: "info"
+      loglevel.activator: "info"
+      loglevel.hpaautoscaler: "info"
+      loglevel.net-certmanager-controller: "info"
+      loglevel.net-istio-controller: "info"
+      loglevel.net-kourier-controller: "info"
+----
++
+.KnativeEventing CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  config:
+    logging:
+      loglevel.controller: "info"
+      loglevel.webhook: "info"
```

Comment:
```suggestion
      loglevel.eventing-webhook: "info"
```

Reply:
We could also include the other eventing components here. The component names (for loglevel configuratin) are sometimes not totally consistent with the name of their deployments, here they are (yes, the mt_broker_filter and ingress has underscores instead of hypens):
```
      loglevel.inmemorychannel-dispatcher: "info"
      loglevel.inmemorychannel-webhook: "info"
      loglevel.mt-broker-controller: "info"
      loglevel.mt_broker_filter: "info"
      loglevel.mt_broker_ingress: "info"
      loglevel.pingsource-mt-adapter: "info"
```